src/constants/files.go

```diff
@@ -2,5 +2,6 @@ package constants
 
 const (
 	ConfigFilePath         = "/.config/scruticode/settings.toml"
+	ReadmeFilePath         = "README.md"
```

Comment:
I'm assuming that this always takes in consideration the current path. Which in the future you should be able to select a path or if empty use the current one.